examples/satellite-azure/README.md

```diff
@@ -155,11 +177,23 @@ module "satellite-host" {
 | cp_hosts                              | A list of Azure host objects used to create the location control plane, including parameters instance_type and count. Control plane count values should always be in multipes of 3, such as 3, 6, 9, or 12 hosts.                  | list   | [<br>&ensp; {<br>&ensp;&ensp; instance_type = "Standard_D4as_v4"<br>&ensp; count         = 3<br>&ensp;&ensp; }<br>]             | yes    |
 | addl_hosts                            | A list of Azure host objects used for provisioning services on your location after setup, including instance_type and count, see cp_hosts for an example.                  | list   | []             | yes    |
 | ssh_public_key                        | SSH Public Key. Get your ssh key by running `ssh-key-gen` command | string   | n/a              | no    |
-
+| cluster_profile                       | Profile information of cluster hosts                              | string   | mx2-8x64| no       |
+| create_cluster                        | Create cluster Disable this, not to provision cluster             | bool     | true    | no       |
```

Comment:
switch true -> false here, and if leaving satellite-ibm variable as false, make sure to update that readme too

Reply:
Updated!